components/contribution-flow/index.js

```diff
@@ -1007,10 +1007,10 @@ class CreateOrderPage extends React.Component {
                       </P>
                     </Flex>
                   </Flex>
-                  {stepDetails.platformFee?.label === 'Other' && (
+                  {stepDetails.platformFee?.label === this.props.intl.formatMessage(messages.platformFeeOther) && (
```

Comment:
Labels are usually intended for display, not computation. We don't have full control on what's added through the translations - we do some basic review, but we could easily miss something - so using them for something else than display can quickly create bugs or even be dangerous.

When possible it's better to use the value, or a special metadata for these kind of checks. For example, when building the option in https://github.com/opencollective/opencollective-frontend/blob/ea86e3c074e1c3bd2713e63b4ac1ef27554e61d5/components/contribution-flow/index.js#L876, you could add a special `isCustomAmount: true` flag.

Reply:
Understood! Will update it 👍